backend/protocol_rpc/endpoints.py

```diff
@@ -398,7 +407,7 @@ def register_all_rpc_endpoints(
     register_rpc_endpoint_for_partial(get_validator, validators_registry)
 
     register_rpc_endpoint_for_partial(get_transaction_by_id, transactions_processor)
-    register_rpc_endpoint_for_partial(get_contract_state, accounts_manager, msg_handler)
```

Comment:
@AgustinRamiroDiaz could you please confirm that you removed this from here because it isn't being used inside the get_contract_data function?

Reply:
you were right! fixed